src/tests/integration/test-data/show/AssertTest.testFail_expect_revert().expected

```diff
@@ -1724,7 +1738,9 @@ module SUMMARY-TEST%ASSERTTEST.TESTFAIL-EXPECT-REVERT():0
              <k>
```

Comment:
From here on, the diff mechanism of GitHub is getting a bit confused because there are many lines that are the same, but then many that are different in between. This is because of the two additional nodes, 13 and 14.